### ISSUE TITLE
Proper syntax of ODESystem unclear

### ISSUE THREAD

**Issue by jd-lara** [SciML/ModelingToolkit.jl#321]
The current example in the documentation 

```Julia
using ModelingToolkit

@parameters t σ ρ β
@variables x(t) y(t) z(t)
@derivatives D'~t

eqs = [D(x) ~ σ*(y-x),
       D(y) ~ x*(ρ-z)-y,
       D(z) ~ x*y - β*z]

de = ODESystem(eqs,[x,y,z],[σ,ρ,β])
```

seems to be broken 

```Julia
julia> de = ODESystem(eqs,[x,y,z],[σ,ρ,β])
ERROR: MethodError: no method matching ODESystem(::Array{Equation,1}, ::Array{Operation,1}, ::Array{Operation,1})
Closest candidates are:
  ODESystem(::AbstractArray{#s28,1} where #s28<:Equation, ::Any, ::Any, ::Any; systems, name) at /Users/jdlara/.julia/packages/ModelingToolkit/8dWrO/src/systems/diffeqs/odesystem.jl:67
  ODESystem(::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any) at /Users/jdlara/.julia/packages/ModelingToolkit/8dWrO/src/systems/diffeqs/odesystem.jl:26
  ODESystem(::Array{Equation,1}, ::Variable, ::Array{Variable,1}, ::Array{Variable,1}, ::Base.RefValue{Array{Expression,1}}, ::Base.RefValue{Array{Expression,2}}, ::Base.RefValue{Array{Expression,2}}, ::Base.RefValue{Array{Expression,2}}, ::Symbol, ::Array{ODESystem,1}) at /Users/jdlara/.julia/packages/ModelingToolkit/8dWrO/src/systems/diffeqs/odesystem.jl:26
  ...
Stacktrace:
 [1] top-level scope at REPL[10]:1
 [2] eval(::Module, ::Any) at ./boot.jl:331
 [3] eval_user_input(::Any, ::REPL.REPLBackend) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.4/REPL/src/REPL.jl:86
 [4] run_backend(::REPL.REPLBackend) at /Users/jdlara/.julia/packages/Revise/C272c/src/Revise.jl:1075
 [5] top-level scope at none:0
```

Is the correct way to specify an ODE system just `ODESystem(eqs)` ? Is this the case for the other systems as well?

**Comment by ChrisRackauckas** [comment]
`de = ODESystem(eqs,t,[x,y,z],[σ,ρ,β])` the `t` was missing.

**Comment by jd-lara** [comment]
Thanks, do you want me to leave it open until the docs are fixed?

**Comment by caryan** [comment]
Closed by #380
